pkg/commands/git_commands/remote_loader.go

```diff
@@ -83,14 +84,13 @@ func (self *RemoteLoader) GetRemotes() ([]*models.Remote, error) {
 func (self *RemoteLoader) getRemoteBranchesByRemoteName() (map[string][]*models.RemoteBranch, error) {
 	remoteBranchesByRemoteName := make(map[string][]*models.RemoteBranch)
 
-	cmdArgs := NewGitCmd("branch").Arg("-r").ToArgv()
-	err := self.cmd.New(cmdArgs).DontLog().RunAndProcessLines(func(line string) (bool, error) {
-		// excluding lines like 'origin/HEAD -> origin/master' (there will be a separate
-		// line for 'origin/master')
-		if strings.Contains(line, "->") {
-			return false, nil
-		}
+	cmdArgs := NewGitCmd("for-each-ref").
+		Arg(fmt.Sprintf("--sort=%s", self.AppState.RemoteBranchSortOrder)).
```

Comment:
I've also considered doing validation, storing a new string enum for `remotebranchsortorder` instead of using the sort key directly is a good safeguard.